kolibri/plugins/learn/assets/src/views/LearnImmersiveLayout.vue

```diff
@@ -315,27 +297,30 @@
               ids: lesson.resources.map(resource => resource.contentnode_id),
             },
           }).then(contentNodes => {
-            return this.progressPromise(contentNodes);
+            const removeSelfFilter = node => node.id !== this.content.id;
+            if (this.$store.getters.isUserLoggedIn) {
+              this.mapSidePanelContentProgress(contentNodes).then(
+                () => (this.viewResourcesContents = contentNodes.filter(removeSelfFilter))
+              );
+            } else {
+              this.viewResourcesContents = contentNodes.filter(removeSelfFilter);
+            }
           });
         });
       },
       /**
        * Prepares a list of content nodes which are children of this.content.parent without
-       * this.content. The prepared nodes have their progress values for the current user
-       * (if there is one) mapped onto each node.
+       * this.content and calls fetchContentNodeProgress when the user is logged in.
        *
        * Largely borrowed from modules/topicsTree/handlers.js
        *
        * @modifies this.viewResourcesContents - Sets it to the progress-mapped nodes
        * @modifies this.nextFolder - Sets the value with this.content's parents next sibling folder
        * if found
+       * @modifies useContentNodeProgress.contentNodeProgressMap (indirectly) if the user
+       * is logged in
        */
       fetchSiblings() {
-        // Guard against this in lesson context
-        if (this.lessonId) {
-          return;
-        }
-
         ContentNodeResource.fetchTree({
           id: this.content.parent,
           params: {
```

Comment:
If you're only interested in the immediate children, can also pass `depth: 1` in the params, to avoid recursing to the grand children of the parent.